features/facebookImport/src/importer/importer-util.js

```diff
@@ -46,7 +51,17 @@ async function readJSONDataArray(dataFileName, dataKey, zipFile, zipId = null) {
         );
     }
 
-    const arrayData = rawData[dataKey];
+    return rawData[dataKey];
+}
+
+async function readJSONDataArray(dataFileName, dataKey, zipFile, zipId = null) {
+    const arrayData = await readJSONDataObject(
+        dataFileName,
+        dataKey,
+        zipFile,
+        zipId
```

Comment:
This is initialized to null above. Any reason why it's not here?

Reply:
In both the declaration of `readJSONDataObject` and `readJSONDataArray` it is initialized with `null`. Or am I missing something?

Reply:
I mean zipId. It's not initialized here, it's initialized to null above.

Reply:
Hmm. There is just a method call that gets `zipId` as the last parameter. Above `zipId ` it is initialized by default with `null` in the signature of the method.